c++/cpp2py/converters/std_array.hpp

```diff
@@ -6,10 +6,10 @@ namespace cpp2py {
   template <typename T, size_t R> struct py_converter<std::array<T, R>> {
     // --------------------------------------
 
-    static PyObject *c2py(std::array<T, R> const &v) {
+    static PyObject *c2py(std::array<T, R> v) {
```

Comment:
Ah wait...
std::array is STACK object, you can not really move it always copy.
What shall we do here ?

Reply:
I agree, this will in fact lead to an additional unnecessary stack copy.
Can we really avoid providing all three overloads in these cases?
```c++
c2py(std::array<T, R> &v)
c2py(std::array<T, R> const &v)
c2py(std::array<T, R> &&v)
```